### PR TITLE
 Add new chapter sync with CalDAV e CardDAV

### PR DESCRIPTION
**Pull request by lucagasparini**

### CODE REVIEW

File: administrator-manual/en/webtop5.rst [hunk 1]
```diff
@@ -96,10 +96,10 @@ To enable 2FA:
 .. image:: _static/webtop-2fa.png 
 
 
-Device synchronization with ActiveSync (EAS)
-============================================
+Synchronization with ActiveSync (EAS)
+=====================================
 
-Devices can be synchronized using ActiveSync.
+Devices mobile can be synchronized using ActiveSync.
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: "Devices mobile" should be "Mobile devices"

File: administrator-manual/en/webtop5.rst [hunk 2]
```diff
@@ -160,7 +160,7 @@ Mobile devices based on Apple iOS fully support folders / categories for calenda
 Mobile devices based on Android instead only support calendars and contacts (activities are not natively supported), 
 but only on the calendars are supported folders / categories, without including colors using the native application Google Calendar.
 
-Installing and using the CloudCal application: https://play.google.com/store/apps/details?id=net.cloudcal.cal&hl=en
+Installing and using the CloudCal application (`CloudCal <https://pselis.com/cloudcal/>`_)
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: Insert the link directly to the first occurrence of "CloduCal":

 Installing and using the `CloudCal <https://pselis.com/cloudcal/>`_ application

File: administrator-manual/en/webtop5.rst [hunk 3]
```diff
@@ -185,6 +185,75 @@ To do so, just right click on the shared resource → Customize → Sync. device
...
+- **User name**: enter your full user name (i.e. *goofy@nethserver.org*)
+- **Password**: enter your password
+
+Some third-party clients allow to simplify the configuration through the *autodiscovery* feature that automatically discovers the 
+synchronizable resources, as in the case of mobile devices clients (i.e. Android or iOS devices).
+
+
+.. note::
+
+   If you are using clients that do not support autodiscovery, you need to use the full URL: ``https://<server_name>/webtop-dav/server.php``
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: If the client supports the autodiscovery, which is the URL?

File: administrator-manual/en/webtop5.rst [hunk 4]
```diff
@@ -185,6 +185,75 @@ To do so, just right click on the shared resource → Customize → Sync. device
...
+
+
+Google Android
+--------------
+
+- install the `Opensync <https://deependhulla.com/android-apps/opensync-app>`_ application from the market;
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: Add a line which introduces the OpenSync as suggested application.

File: administrator-manual/en/webtop5.rst [hunk 5]
```diff
@@ -185,6 +185,75 @@ To do so, just right click on the shared resource → Customize → Sync. device
...
+- add a new account clicking on "``+``" key and select :guilabel:`Login with URL and username` method;
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: Use guilabel tag got the "+"

File: administrator-manual/en/webtop5.rst [hunk 6]
```diff
@@ -185,6 +185,75 @@ To do so, just right click on the shared resource → Customize → Sync. device
...
+- insert the ``URL`` (``https://<server_name>``), complete username (i.e. *goofy@nethserver.org*) and password;
+- click on the new profile and select the resources you want to synchronize.
+
+Apple iOS
+---------
+
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: Add an introduction line, something like:
"CalDav/CardDav support is built-in on iOS" ...

File: administrator-manual/en/webtop5.rst [hunk 7]
```diff
@@ -185,6 +185,75 @@ To do so, just right click on the shared resource → Customize → Sync. device
...
+- Settings -> Account and Password -> Add account;
+- select :guilabel:`Other` -> Add :guilabel:`CalDAV` or :guilabel:`CardDAV` account;
+- insert the server name (i.e. *server.nethserver.org*), complete username (i.e. *goofy@nethserver.org*) and password.
+
+.. note::
+
+   At the moment mobile devices support **only personal resources synchronization**.
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: Does it mean that Thunderbird support even shared resources?

File: administrator-manual/en/webtop5.rst [hunk 8]
```diff
@@ -185,6 +185,75 @@ To do so, just right click on the shared resource → Customize → Sync. device
...
+ 
+By default the syncronization ``URL`` uses the server principal name (``FQDN``), if you need to change it: ::
+
+ config setprop webtop DavServerUrl https://<new_name_server>/webtop-dav/server.php
+ signal-event nethserver-webtop5-update
+
+
+Tips for some desktop clients
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: Change to
"Desktop clients"

Add an introduction for desktop client, for example that you need to install CardBook and Lightning in TB.

File: administrator-manual/en/webtop5.rst [hunk 9]
```diff
@@ -185,6 +185,75 @@ To do so, just right click on the shared resource → Customize → Sync. device
...
+-----------------------------
+
+**Thunderbird**
+
+- Lightning doesn't support autodiscovery: any calendar must be manually added.
+- Thunderbird has no native CardDAV support: the ``Cardbook`` add-on works fine, with easy setup and autodiscovery support.
+
+**Outlook**
+
+- opensource ``Outlook CalDav Synchronizer`` client works fine, supporting both CardDAV and CalDAV.
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: "opensource" is misspelled

**Comment by gsanchietti** [comment]
Moved to #344 so @filippocarletti have direct access rights.

File: administrator-manual/en/webtop5.rst (hunk 3 above)
Comment (lucagasparini) [reply administrator-manual/en/webtop5.rst]: Indication added to line 212

**Pull request closed without merging**